addons/web_editor/static/src/js/widgets/widgets.js

```diff
@@ -34,7 +34,7 @@ Dialog = Dialog.extend({
         options = options || {};
         this._super(parent, _.extend({}, {
             buttons: [
-                {text: options.save_text || _t("Save"), classes: 'btn-primary', click: this.save},
+                {text: options.save_text || _t("Add"), classes: 'btn-primary', click: this.save},
```

Comment:
I doubt that the default text for all dialog primary buttons should become "Add" :/

Reply:
Then I'll leave 'Save' as this is nonsense to do that

Reply:
The point is not to leave "Save". It is to use "Add", *only* for the media dialog not all dialogs of Odoo.